ujenkins/endpoints/builds.py

```diff
@@ -21,10 +21,39 @@ class Builds:
     def __init__(self, jenkins) -> None:
         self.jenkins = jenkins
 
-    def get(self, name: str) -> List[dict]:
+    def get(self, name: str,
+            fields: Optional[Union[List[str], str]] = None,
```

Comment:
let fields be just list, not union of str or list

Reply:
I need to use a new variable instead of shadowing the old one
```
fields_str = ','.join(fields) if fields else 'number,url'
```

Reply:
I did not get your point, what is problem to use write `get(self, fields: Optional[List[str]], ...)` ?